diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1c58498..7676b2a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,7 +59,7 @@ trojita_option(WITH_DBUS "Build with DBus library" AUTO)
 trojita_option(WITH_RAGEL "Build with Ragel library" AUTO)
 trojita_option(WITH_ZLIB "Build with zlib library" AUTO)
 trojita_option(WITH_SHARED_PLUGINS "Enable shared dynamic plugins" ON)
-trojita_option(WITH_TESTS "Build tests" ON)
+trojita_option(WITH_TESTS "Build tests" OFF)
 trojita_option(WITH_MIMETIC "Build with client-side MIME parsing" AUTO)
 trojita_option(WITH_GPGMEPP "Build with the GpgME++ library for cryptography" AUTO)
 
